diagnostics/rebuildregkeys.ps1

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for license information.
+
+<#
+Script to rebuild the WSL2 distros regkeys when they are missing from user's registry but provided in Get-AppxPackage
```

Comment:
Why is this WSL2 only? Can support be added for WSL1?

Reply:
no. WSL1 is complex than this, and this script only rebuild for wsl2